.github/workflows/build.yml

```diff
@@ -43,25 +46,38 @@ jobs:
       - uses: docker/setup-buildx-action@v2
 
       - id: vars
-        name: Compute Image Tag
+        name: Compute vars
+        env:
+          ACTIVEMQ_BASE_VERSION: ${{ matrix.activemq_base_version }}
         run: |
-          ACTIVEMQ_VERSION=$(jq -r .activemq_version versions/activemq-${{ matrix.activemq_base_version }}.json)
+          ACTIVEMQ_VERSION=$(jq -r .activemq_version versions/activemq-${ACTIVEMQ_BASE_VERSION}.json)
           echo "activemq_version=$ACTIVEMQ_VERSION" >> $GITHUB_OUTPUT
+          echo "image_created=$(date -u +'%Y-%m-%dT%H:%M:%SZ')" >> $GITHUB_OUTPUT
 
-          TAG_BASE_NAME="$ACTIVEMQ_VERSION-${{ matrix.jdist }}${{ matrix.java_major }}-${{ matrix.base_image.flavor }}${{ matrix.base_image.major }}"
+          QUAY_IMAGE_NAME="quay.io/${IMAGE_REGISTRY_NAMESPACE}/${IMAGE_REPOSITORY}"
+          DOCKERHUB_IMAGE_NAME="${IMAGE_REGISTRY_NAMESPACE}/${IMAGE_REPOSITORY}"
+          TAG_COMMON_METADATA="${{ matrix.jdist }}${{ matrix.java_major }}-${{ matrix.base_image.flavor }}${{ matrix.base_image.major }}"
 
-          if [[ "${{ github.ref_name }}" != "master" ]]
-          then
-            echo "image_tag=${TAG_BASE_NAME}-${GITHUB_REF_NAME//\//-}" >> $GITHUB_OUTPUT
+          if [[ "${{ github.event_name }}" == "pull_request" ]]; then
             echo "labels=quay.expires-after=1d" >> $GITHUB_OUTPUT
-          elif [[ "${{ github.event.head_commit.message }}" =~ .*\[release\].* ]]
-          then
-            echo "image_tag=$TAG_BASE_NAME" >> $GITHUB_OUTPUT
+
+            IMAGE_TAG_PR="${ACTIVEMQ_VERSION}-${TAG_COMMON_METADATA}-pr-${{ github.event.pull_request.number}}"
+            echo "tags=${QUAY_IMAGE_NAME}:${IMAGE_TAG_PR}" >> $GITHUB_OUTPUT
           else
-            echo "image_tag=${TAG_BASE_NAME}-release-candidate" >> $GITHUB_OUTPUT
+            IMAGE_TAG_SEMVER="${ACTIVEMQ_VERSION}-${TAG_COMMON_METADATA}"
+            IMAGE_TAG_MINOR="${ACTIVEMQ_BASE_VERSION}-${TAG_COMMON_METADATA}"
+
+            echo "tags<<EOF" >> $GITHUB_OUTPUT
```

Comment:
I assumed because they are calling an external service which could return the same EOF string you chose. In this case I hope now one is going to release activemq vEOF 😀